install.sh

```diff
@@ -32,50 +32,52 @@ mv vc/tools/MSVC vc/tools/msvc
 # /DEFAULTLIB:"OLDNAMES", which lld-link doesn't find on a case sensitive
 # filesystem. Therefore add matching case symlinks for this, to allow
 # linking MSVC built objects with lld-link.
-cd $(echo vc/tools/msvc/* | awk '{print $1}')/lib
+cd "$(echo vc/tools/msvc/* | awk '{print $1}')/lib"
 for arch in x86 x64 arm arm64; do
-    cd $arch
+    cd "$arch"
     for i in libcmt libcmtd msvcrt msvcrtd oldnames; do
-        ln -s $i.lib $(echo $i | tr [a-z] [A-Z]).lib
+        ln -s "$i.lib" "$(echo "$i" | tr '[a-z]' '[A-Z]').lib"
```

Comment:
Also unnecessary

Reply:
True. Thing is my editor's linter even complains about unquoted strings anywhere, so I just mechanically add them. :smile: